core/services/claim_workflow.py

```
@@ -397,26 +397,38 @@ def process_claim_payment(self, claim_id: str, payment_data: Dict[str, Any]) ->
+        from core.services.notification_system import NotificationServiceFactory
+        self.notification_service = NotificationServiceFactory.create_notification_service()
+    
     def notify_claim_submitted(self, claim_id: str) -> None:
         """Notify stakeholders of claim submission"""
-        # TODO: Implement actual notification logic
-        # This could include email, SMS, in-app notifications, etc.
-        print(f"Notification: Claim {claim_id} has been submitted")
+        try:
+            self.notification_service.notify_claim_submitted(claim_id)
+        except Exception as e:
+            logger.exception(f"Failed to send claim submitted notification for {claim_id}: {e}")
```

Ruff (0.13.2)

Undefined name `logger`

(F821)